app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -149,8 +149,9 @@ def find_host_by_provider_id(provider_id)
     def scheme_to_provider_mapping
       @scheme_to_provider_mapping ||= begin
         {
-          'gce' => ['ManageIQ::Providers::Google::CloudManager'.safe_constantize, :name],
-          'aws' => ['ManageIQ::Providers::Amazon::CloudManager'.safe_constantize, :uid_ems],
+          'gce'       => ['ManageIQ::Providers::Google::CloudManager'.safe_constantize, :name],
+          'aws'       => ['ManageIQ::Providers::Amazon::CloudManager'.safe_constantize, :uid_ems],
+          'openstack' => ['ManageIQ::Providers::Openstack::CloudManager'.safe_constantize, :uid_ems]
```

Comment:
I think this bug happend because OpenStack changed something. This means that we want to keep the old part for backward compatibility sake..